templates/catalogs.html.twig

```diff
@@ -0,0 +1,48 @@
+{% extends 'connected-base.html.twig' %}
+
+{% block title %}{{ 'page.products.title' | trans }}{% endblock %}
+
+{% block content %}
+    <div class="catalogs">
+        {% embed './commons/information.html.twig' with {helperTitle: 'page.catalogs.information.helper-title'|trans } %}
+            {%  block informationContent %}
+                <p>{{ 'page.catalogs.information.content-1' | trans }}</p>
```

Comment:
Having the whole message inside a single key may line break mid-sentence and is visually ugly IMO